ansible/setup.yml

```diff
@@ -93,6 +57,27 @@
         - ssh-keys
         - deploy-ssh-keys
 
+    - name: Download Vault public key
+      get_url:
+        url: "{{ vault_addr }}/v1/ssh-client-signer/public_key"
+        dest: /etc/ssh/trusted-user-ca-keys.pem
+      when: vault_addr is defined
+      tags:
+        - ssh-keys
+        - developer-ssh-keys
```

Comment:
This task is only updating the CA cert - not the users. Right?
If I understand correctly, the user SSH-ing is authorized based on GitHub token, that is associated to GitHub account. So it means that user pubkeys on GitHub are not relevant any longer, provided that user provides GitHub token (whether GitHub invalidates tokens when pubkeys changes, that is a separate independent question GitHub-specific). So it means that the list of users is not kept here but in vault (kept and edited directly in vault manually by admin), so there is no need any longer to poll user pubkeys - or any other user info - from GitHub, right?

Reply:
Your got it right at 100% !